package.json

```diff
@@ -18,16 +18,23 @@
     "ec": "./dist/index.js",
     "editorconfig-checker": "./dist/index.js"
   },
+  "publishConfig": {
+    "access": "public",
+    "provenance": true
+  },
   "files": [
     "dist"
   ],
   "scripts": {
-    "lint": "eslint . --fix",
+    "lint:eslint": "eslint . --max-warnings 0 --report-unused-disable-directives --ignore-path .gitignore",
```

Comment:
That's amazing - I did not know about the `--ignore-path` flag 🙏

Reply:
Yeah, I find that way nicer than a `.eslintignore` there are already too much dot files in our project. :laughing:
Since Prettiver v3, `.gitignore` is the default, but not yet for ESLint unfortunately, so we must specify it.

This change is unrelated to the main change of this PR of course, but as I was editing the lint commands, might just do that aswell. :smile: